addons/mail/static/src/discuss/core/web/burger_menu_patch.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">  
+    <t t-inherit="web.BurgerMenu" t-inherit-mode="extension">
+        <xpath expr="//img[hasclass('o_burger_menu_avatar')]" position="replace">
+            <div class="position-relative d-flex flex-shrink-0 bg-inherit">
+                <t>$0</t>
+                <ImStatus className="'o-web-UserMenu-imStatus position-absolute'" persona="store.self" size="'md'"/>
```

Comment:
Would add chevron to give a clue this is clickable to open another menu:
<img width="322" height="544" alt="Screenshot 2025-09-03 at 13 29 10" src="https://github.com/user-attachments/assets/e7bdfda4-c7d5-4688-969a-2c75527fc4f7" />

So the visual matches the desktop version:
<img width="194" height="258" alt="Screenshot 2025-09-03 at 13 31 37" src="https://github.com/user-attachments/assets/cab697e9-6532-4e8e-968f-f6db62c258ad" />

The other items don't need that because they are named as menu action. The online part looks like just current status and without chevron / caret this is not made obvious we can interact with it.